packages/worker/src/worker-options.ts

```diff
@@ -407,11 +408,11 @@ export interface WorkerOptions {
   /**
    * A mapping of interceptor type to a list of factories or module paths.
    *
-   * By default, {@link ActivityInboundLogInterceptor} and {@link WorkflowInboundLogInterceptor} are installed. If you
-   * wish to customize the interceptors while keeping the defaults, use {@link appendDefaultInterceptors}.
-   *
    * When using {@link workflowBundle}, these Workflow interceptors (`WorkerInterceptors.workflowModules`) are not used.
    * Instead, provide them via {@link BundleOptions.workflowInterceptorModules} when calling {@link bundleWorkflowCode}.
+   *
+   * The SDK itself may automatically append interceptors required to support its own features Explicitely calling
```

Comment:
typo explicitly
Does the order matter, i.e., the interceptors added are run before or after the user provided ones?

Reply:
Yes, ordering matters. Interceptors are called in order, from the first to the last, each one making the call to the next one, and the last one calling the original function. I'll add this bit of info to the doc.

Reply:
Now, that leaves us with a minor problem: until now, `appendDefaultInterceptors()` was _appending_ our log interceptors. In this PR, I retained that ordering, but I think it is wrong. Built-in interceptors should be prepended, so that they capture what happens inside user's interceptors. For example, if the workflow function completed succesfully, but a custom interceptor wrapping it throws an exception, I'd say that WorkflowLogInterceptor should report that exception, rather than success.

I'm therefore changing this to _prepending_ internal interceptors instead. That might slightly change SDK's observable behaviors, which could techically be breaking in some rare cases, but I feel ok with that one. @bergundy WDYT?

Reply:
Yeah.. I think prepending is fine if we want to capture the durations and alterations made by user interceptors.
For the default log attributes though we should ensure that users can modify our defaults.

Reply:
Of course. That is what will happen by prepending internal interceptor.